source/basic.tex

```diff
@@ -5145,8 +5145,8 @@
 \indextext{type!integral}%
 Types
 \tcode{bool},
```

Comment:
What's the intrinsic? Why not all `\tcode{`_`X`_`}`, where _`X`_ is a [_keyword_](http://wg21.link/lex.key#nt:keyword)?

Reply:
All keywords should be marked `\keyword`. This way, we can make them a different color sometime in the future. You also get an index entry.